Jantimizer/QueryPlanParser/Caches/QueryPlanCacher.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Tools.Caches;
+
+namespace QueryPlanParser.Caches
+{
+    public class QueryPlanCacher : ICacherService<ulong?>
+    {
+        public static QueryPlanCacher? Instance { get; set; }
+
+        internal Dictionary<string, ulong?> ActualCardinalityCache { get; set; }
```

Comment:
Is nullable<ulong> preferable to not adding the value to the dictionary, if it's null?